readme.md

```diff
@@ -306,6 +306,7 @@
 	- [Spanish](https://github.com/dav009/awesome-spanish-nlp#readme)
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby#readme)
 	- [Question Answering](https://github.com/seriousmac/awesome-qa#readme) - The science of asking and answering in natural language with a machine.
+	- [Natural Language Generation](https://github.com/tokenmill/awesome-nlg) - Automated creation of natural language text, used to convert data to text, response generation for conversational agents, or create narratives for reports, news items and a like.
```

Comment:
```suggestion
	- [Natural Language Generation](https://github.com/tokenmill/awesome-nlg#readme) - Automated creation of natural language text, used to convert data to text, response generation for conversational agents, or create narratives for reports, news items and a like.
```

Reply:
Would be great if you could try to shorten the description. It's a bit too verbose.